### DISCUSSION TITLE
Training stops at 500 steps, despite 1700 max steps

### DISCUSSION THREAD

**Discussion by wktra** [d8ahazard/sd_dreambooth_extension#408] [General]
Anyone else getting this error?

![Screenshot 2022-12-03 214239](https://user-images.githubusercontent.com/117292904/205473234-34eb7d1e-6daf-428b-a897-3a8df7c366c8.png)

**Comment by AlexMcCown** [comment]
It looks like a weeks old bug is appearing again, change your 'Save Preview(s) Frequency' to a number outside the max steps 
it appears to be crashing while generating a sample image and they only _really_ act as progress reports
